packages/office-ui-fabric-react/src/utilities/dateMath/DateMath.ts

```diff
@@ -198,4 +198,46 @@ function getDatePartHashValue(date: Date) {
   /* tslint:disable:no-bitwise */
   return date.getDate() + (date.getMonth() << 5) + (date.getFullYear() << 9);
   /* tslint:enable:no-bitwise */
-}
\ No newline at end of file
+}
+
+/**
+ * Returns week number for a week in a year.
+ */
+export function getWeekNumbers(weeks: any[], firstDayOfWeek: DayOfWeek, navigatedDate: Date) {
+  let monthRange = getDateRangeArray(navigatedDate, DateRangeType.Month, firstDayOfWeek);
+
+  let getDayofYear = getDayOfYear(monthRange[0]);
```

Comment:
Do you really need to fetch the entire month range just to get the first day of the month? Can you just do new Date(navigatedDate.getYear(), navigatedDate.getMonth(), 1)?